src/components/FileInput.js

```
@@ -0,0 +1,56 @@
+      this.input.multiple = true;
+    }
+
+    this._handleChange = this._handleChange.bind(this);
+    this.input.addEventListener('change', this._handleChange);
+
+    this.parent.appendChild(this.input);
+  }
+
+  _handleChange(event) {
+    const { files } = event.target;
+    if (files && files.length && typeof this.onFileSelected === 'function') {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)